pages/positions/options.tsx

```diff
@@ -21,13 +21,13 @@ const Options: NextPage = () => {
       <div className={style.optionsContainer}>
         <h1 className={style.optionsHeader}>Open a New Position</h1>
         <div className={style.optionsButtonContainer}>
-          <Link href={'addliquidity'}>
+          <Link href={'deposit/0'}>
```

Comment:
where does this link go? does it still go to `addLiquidity`

Reply:
yes, the address will be 0, it needs to have some value